lib/elastic_apm/spies/net_http.rb

```diff
@@ -36,30 +38,49 @@ def request(req, body = nil, &block)
             unless (transaction = ElasticAPM.current_transaction)
               return request_without_apm(req, body, &block)
             end
+
             if ElasticAPM::Spies::NetHTTPSpy.disabled?
               return request_without_apm(req, body, &block)
             end
 
-            host, = req['host']&.split(':')
-            method = req.method
+            host = req['host']&.split(':')&.first || address
```

Comment:
Apologies for my Ruby ignorance: what's the content of `req['host']` when IPv6 is used? Where's this `address` variable come from?

That splitting looks suspiciously like it won't work for IPv6, but I see you have a test for net_http and IPv6. Does it cover all branches?

Reply:
Comes from all the way back in the Opbeat days: https://github.com/opbeat/opbeat-ruby/blob/master/lib/opbeat/injections/net_http.rb#L14 !
I'm not actually really sure it's set anymore. `address` is a method on the object we're currently prying ourselves into (yay Ruby!).